### ISSUE TITLE
API to get list of series fails

### ISSUE THREAD

**Issue by phagunbaya** [grafana/grafana#1603]
Grafana sends API like http://localhost:8086/db/sampledb/series?p=root&u=root
But influx expects it to be http://localhost:8086/db/sampledb/series?p=root&u=root&q=list%20series
Note - We are on influx 0.8.8

Is there anything we can get pass this ?

**Comment by torkelo** [comment]
strange, so when you type in letters in the series name you do not get any suggestions? Works for me, what Grafana version are you using?

**Comment by phagunbaya** [comment]
Yes, it does not give any suggestions. Grafana 1.9.1

**Comment by torkelo** [comment]
Strange, no one else has reported problems with this feature with Grafana 1.9.1, can you check chrome dev tools as you type in your series name (I think you need at least 3 characters before the auto complete kicks in). what OS & Browser are you using?

**Comment by phagunbaya** [comment]
OK. Now I got it. It is working fine in now. I encountered an error where it continuously sends request to fetch list of series without "q" param. This issue can be closed now.

Other question is, if none of the name matches the list of series, I see "list_series_list" in the suggestion list.
![screen shot 2015-03-18 at 6 27 45 pm](https://cloud.githubusercontent.com/assets/1146297/6709135/86249d9e-cd9c-11e4-8dc1-ed7c885d0380.png)


Thanks for the help !
